lib/controller.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const _ = require('lodash');
```

Comment:
Actually, maybe it is in scope... having `underscore` *and* `lodash` in the same project is N.A.S.T.Y.